### ISSUE TITLE
Modify the backbone network

### ISSUE THREAD

**Issue by yuyu19970716** [JUGGHM/PENet_ICRA2021#54]
Hi!
I have a question for you!
Recently I changed PENet's backbone network ENet to attention-unet, and I only used part of Kitti's dataset. I felt that the training was a little slow, and then I modified the learning rate when training the backbone network so that the network could learn faster.
![image](https://user-images.githubusercontent.com/76560812/173505009-9ced8db3-ddf4-4f0a-84b7-e31b31da9a2f.png)
It's ok when I train ENet. (Blue is the training set, yellow is the validation set)
But when I used the trained backbone network for the second stage of training, the network experienced severe overfitting. I wonder if this has something to do with the learning rate?
![image](https://user-images.githubusercontent.com/76560812/173505199-e6953445-1877-4542-9de4-983f53646886.png)
We can see that cspn++ is trained very poorly. The validation set error is large.
What is the reason for this?
Below is the learning rate when I train ENet.
![f5519c1a5763a78fb707e811817d022](https://user-images.githubusercontent.com/76560812/173505426-1084b7ad-333f-473f-9c70-53cbf05d1419.png)

**Comment by yuyu19970716** [comment]
![image](https://user-images.githubusercontent.com/76560812/173505812-4ab68b63-5e75-43bb-9728-94aa55b997c6.png)
The above is the result after training ENet
I am very much looking forward to your reply!

**Comment by JUGGHM** [comment]
It seems reasonable. Since the stage2 training should be regarded as an parametric initialization step of the CSPN++ module, it is a normal case that error metrics after stage2 are no better than those after stage1. You could refer to previous issues for detailed discussion: issue [30](https://github.com/JUGGHM/PENet_ICRA2021/issues/30) [28](https://github.com/JUGGHM/PENet_ICRA2021/issues/28) [25](https://github.com/JUGGHM/PENet_ICRA2021/issues/25).

**Comment by yuyu19970716** [comment]
Thank you very much for your prompt answer!
I will seriously consider your proposal!

**Comment by yuyu19970716** [comment]
![image](https://user-images.githubusercontent.com/76560812/173742337-bdb5aa71-eb40-4c60-95ec-073771ee6842.png)

But I independently verified the second stage in the code you posted, and the validation and training sets fit just fine. Why is this?

**Comment by yuyu19970716** [comment]
Could it be because I have too little data? I only downloaded the dataset for some of the dates instead of all dates, and now I'm adding the datasets of other dates to train again.
